### PR TITLE
Verification Id added in certificate

### LINKED ISSUE #20: [Feature Request]: Add a unique Id in the certificate for verification.
### Preflight Checklist

- [X] I have read the Contributions.md for this project.
- [X] I agree to follow the Code of Conduct that this project adheres to.
- [X] I have searched the feature for a feature request that matches the one I want to file, without success.

### Problem Description

For validating the authenticity of the certificate there is already a QR code in the certificate but for a more handly way for quick validation, we can also give unique IDs printed on the certificates, for one-click verification, on the website.

### Proposed Solution

Add a unique ID on the certificate maintaining the proper design, unique ID can be the Object ID of the certificate itself.

### Alternatives Considered

NA

### Additional Information

NA

### PR DESCRIPTION
**Pull request by Simran2337**
# Description:

Verification ID displayed on Certificate

Fixes #20

## Type of change:

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

# Screenshots / Video:
image

### CODE REVIEW

File: src/views/certificate.ejs [hunk 1]
```diff
@@ -39,7 +39,13 @@
             style = "height: 100% ;width: 100% ;padding: 0px;margin: 0px;	" > 
         </img> 
         <div style = "position: absolute;top: 0;left: 0;height: 1155px;width: 1500px;padding: 0px;margin: 0px;">
-            <div style="position: absolute; right: 50px; margin: 10px; color: rgb(224 68 44); font-family: monospace;">Certified On: <%= data.createdAt.toISOString().split('T').join(' ').slice(0,19) %></div>
+            <div
```
Comment (Shubham-Kumar-2000) [inline src/views/certificate.ejs]: - Put this in bottom left.
- Make ID readable like `67vj7-sdkvb-skvjs-sdkjv` add - after every 5 characters.

File: src/views/certificate.ejs [hunk 2]
```diff
@@ -39,7 +39,13 @@
             style = "height: 100% ;width: 100% ;padding: 0px;margin: 0px;	" > 
         </img> 
         <div style = "position: absolute;top: 0;left: 0;height: 1155px;width: 1500px;padding: 0px;margin: 0px;">
-            <div style="position: absolute; right: 50px; margin: 10px; color: rgb(224 68 44); font-family: monospace;">Certified On: <%= data.createdAt.toISOString().split('T').join(' ').slice(0,19) %></div>
+            <div
+                style="position: absolute; right: 50px; margin: 10px; margin-top:6.5px; color: rgb(224 68 44); font-family: monospace;">
+                Id: <%= data._id %>
```
Comment (Shubham-Kumar-2000) [inline src/views/certificate.ejs]: instead of `ID` use `Certificate Id` its more intent revealing.

**Review by Shubham-Kumar-2000 — CHANGES_REQUESTED** [review]

**Comment by Simran2337** [comment]
image


@Shubham-Kumar-2000  changes made.
Can u please check and revert back?

File: src/views/certificate.ejs [hunk 3]
```diff
@@ -38,8 +38,9 @@
         <img src = "https://user-images.githubusercontent.com/41825906/153724469-5b2f41be-e317-4ba8-88fb-b2832aab507a.png"
             style = "height: 100% ;width: 100% ;padding: 0px;margin: 0px;	" > 
         </img> 
-        <div style = "position: absolute;top: 0;left: 0;height: 1155px;width: 1500px;padding: 0px;margin: 0px;">
-            <div style="position: absolute; right: 50px; margin: 10px; color: rgb(224 68 44); font-family: monospace;">Certified On: <%= data.createdAt.toISOString().split('T').join(' ').slice(0,19) %></div>
+        <div style = "position: absolute;top: 0;left: 0;height: 1155px;width: 1500px;padding: 0px;margin: 0px;">   
+            <div style="position: absolute; right: 50px; margin: 10px; margin-top: 23px; color: rgb(224 68 44); font-family: monospace;">
```
Comment (Shubham-Kumar-2000) [inline src/views/certificate.ejs]: Remove the extra top margin.

**Review by Shubham-Kumar-2000 — CHANGES_REQUESTED** [review]

**Review by Shubham-Kumar-2000 — APPROVED** [review]

**Pull request merged**